Dockerfiles/agent/Dockerfile

```diff
@@ -211,6 +209,12 @@ RUN  mv /etc/s6/init/init-stage3 /etc/s6/init/init-stage3-original
 COPY init-stage3          /etc/s6/init/init-stage3
 COPY init-stage3-host-pid /etc/s6/init/init-stage3-host-pid
 
+# Update if optional OTel Agent process should not run
+RUN if [ ! -f /opt/datadog-agent/embedded/bin/otel-agent ]; then \
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does it work ? The syntax should be `if ! [ -f ... ]; then ...`

Reply:
Hey @CelianR actually the more portable way to negate the test is with the `!` inside the test `[]`. See https://unix.stackexchange.com/questions/314486/shell-logical-not.